source/webapp/sites/all/modules/custom/checkbook_project/includes/checkbook_project.inc

```diff
@@ -1969,7 +1969,7 @@ function _checkbook_append_url_params($source = null, $overidden_params = array(
     if (($current_url[3] == 'contract' && ($current_url[4] == 'search' || $current_url[4] == 'all') && $current_url[5] == 'transactions')) {
       $advanced_search = true;
     }
-    if (!$advanced_search) {
+    if (isset($advanced_search) && !$advanced_search) {
```

Comment:
Gives undefined variable error, and adding this check won't impact the current logic/program flow.